src/config.js

```diff
@@ -0,0 +1,21 @@
+const hasEnvVariables = typeof process !== 'undefined' && typeof process.env !== 'undefined'
+
+function env(key) {
+  return hasEnvVariables ? process.env[key] : undefined
```

Comment:
Ouh, I'm afraid this will not work well with webpack Define plugin.

Simply, there's no such thing as `process.env` on client (browser). But the common way to provide such information (upon build) is just to replace all strings `process.env.NODE_ENV` (or any which you define) with some value - in this case, `"development"` or `"production"`.

However, verify this please. I may be wrong - did you try this build out ?

Reply:
Yes, I've tried it with webpack Define plugin, seems to work ok.